Admin/Client/src/authentication/index.tsx

```diff
@@ -0,0 +1,24 @@
+import SideEffectJs from 'side-effect-js';
+import { LoginSideEffectType } from '../sideEffects/authentication/loginSideEffect';
+export default class Authentication {
+    private static _token: string = '';
+
+    public static IsAuthenticated() {
```

Comment:
maybe use some cookie?
since we do not have any tokenization logic

Reply:
Cookie / Local storage - this is the next step - currently it saves in memory